<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>470</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_1">
      <property name="text">
       <string>시험지 만들기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_3">
      <property name="text">
       <string>시험지 인쇄하기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_4">
      <property name="text">
       <string>답지 보기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_quit">
      <property name="text">
       <string>나가기</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>200</y>
     <width>291</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Set days (1~40)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="set_day">
      <property name="inputMethodHints">
       <set>Qt::ImhPreferNumbers</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="set_eng">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhPreferNumbers</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Set English(0~30)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Set Korean(0~30)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="set_kor">
      <property name="inputMethodHints">
       <set>Qt::ImhPreferNumbers</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>811</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
  </widget>
  <zorder>table</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>gridLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
